google-search.css

```diff
@@ -1,110 +1,141 @@
-  #cse-container {
+#cse-container {
     width: 75%;
   }
-  #cse {
+#cse {
   	width: 100%;
   }
-  #cse-search-form {
+#cse-search-form {
   	width: 30%;
   }
   
-  
-  .gsc-control-cse {
+.gsc-control-cse {
     font-family: Arial, sans-serif;
     border-color: #FFFFFF;
     background-color: #FFFFFF;
   }
-  input.gsc-input {
+
+input.gsc-input {
     border-color: #A25B08;
   }
-  td.gsc-search-button {
+
+td.gsc-search-button {
     border-left: 5px solid white;
   }
-  input.gsc-search-button {
+
+input.gsc-search-button {
     border-color: #461200;
     background-color: #461200;
   }
-  .gsc-tabHeader.gsc-tabhInactive {
+
+.gsc-tabHeader.gsc-tabhInactive {
     border-color: #A25B08;
     background-color: #A25B08;
   }
-  .gsc-tabHeader.gsc-tabhActive {
+
+.gsc-tabHeader.gsc-tabhActive {
     border-color: #461200;
     background-color: #461200;
   }
-  .gsc-tabsArea {
+
+.gsc-tabsArea {
     border-color: #461200;
   }
-  .gsc-webResult.gsc-result {
+
+.gsc-webResult.gsc-result {
     border-color: #FFFFFF;
     background-color: #FFFFFF;
   }
-  .gsc-webResult.gsc-result:hover {
+
+.gsc-webResult.gsc-result:hover {
     border-color: #FFFFFF;
     background-color: #FFFFFF;
   }
-  .gs-webResult.gs-result a.gs-title:link,
-  .gs-webResult.gs-result a.gs-title:link b {
+
+.gs-webResult.gs-result a.gs-title:link,
+
+.gs-webResult.gs-result a.gs-title:link b {
     color: #950000;
   }
-  .gs-webResult.gs-result a.gs-title:visited,
-  .gs-webResult.gs-result a.gs-title:visited b {
+
+.gs-webResult.gs-result a.gs-title:visited,
+
+.gs-webResult.gs-result a.gs-title:visited b {
     color: #950000;
   }
-  .gs-webResult.gs-result a.gs-title:hover,
-  .gs-webResult.gs-result a.gs-title:hover b {
+
+.gs-webResult.gs-result a.gs-title:hover,
+
+.gs-webResult.gs-result a.gs-title:hover b {
```

Comment:
I am happy with indentation changes, but extra blank lines here seem to make things less clear here, not more?